src/content/en/rules/index.md

```diff
@@ -54,7 +56,9 @@ The following are also factors in the payout:
 
 * If you provide a code fix for the vulnerability then you will also be eligible for a "code fix" bonus on the condition that our existing developers accept it as valid.
 
-All payouts will be in Decred only. Payouts are done in a single batch once a month. You will be required to create and operate a Decred wallet. The DCR to USD ratio is based on the the average USD rate of the previous month. Payout amount is decided by a core "bug bounty" group.
+All payouts will be in Decred only. Payouts are done in a single batch once a month. You will be required to create and operate a Decred wallet and CMS account. The DCR to USD ratio is based on the average USD rate of the previous month. The payout amount is decided by a core "bug bounty" group. Once decided unless the vulnerability impact changes there will be no changes in the payout amount.
+
+For certain vulnerabilities, we might choose to only release 60% of the payout amount first and hold the 40% until a patch has been released and distributed.
```

Comment:
Can we provide some detail here? "Certain vulnerabilities" is really vague, and even speaking as somebody who helps to run the bounty program, Im not entirely sure what it means

Reply:
Fixed